src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/Type.cs

```diff
@@ -585,16 +585,6 @@ public PredefinedType getPredefType()
             return getAggregate().GetPredefType();
         }
 
-        ////////////////////////////////////////////////////////////////////////////////
-        // Is this type System.TypedReference or System.ArgIterator?
-        // (used for errors because these types can't go certain places)
-
-        public bool isSpecialByRefType()
-        {
-            // ArgIterator, TypedReference and RuntimeArgumentHandle are not supported.
```

Comment:
We've gone back and forth on whether these would be supported in .NET Core, which makes me think that maybe one or more of them would be in the future, which makes me a bit nervous about removing the (theoretically) one place these would need to be reinserted.  And, would this be necessary to properly support span?